src/axom/primal/operators/winding_number.hpp

```diff
@@ -211,7 +212,7 @@ int winding_number(const Point<T, 2>& R,
  * that constructs a polygon with the same *integer* WN as 
  * the curve closed with a linear segment. The *generalized* WN 
  * of the closing line is then subtracted from the integer WN to
- * return the GWN of the original curve. (See )
+ * return the GWN of the original curve.
```

Comment:
nitpick: For clarity, maybe worth it to state the full "generalized winding number"  non-abbreviated form at the top of the file somewhere in the comments, as done in `winding_number.hpp`: https://github.com/LLNL/axom/blob/e5b5707ed09eeb46265b5740a810242970eaaf76/src/axom/primal/operators/winding_number.hpp#L9